packages/eslint-plugin-svelte/src/rules/no-add-event-listener.ts

```diff
@@ -0,0 +1,74 @@
+import type { TSESTree } from '@typescript-eslint/types';
+
+import { createRule } from '../utils/index.js';
+import type { SuggestionReportDescriptor } from '../types.js';
+
+export default createRule('no-add-event-listener', {
+	meta: {
+		docs: {
+			description: 'Warns against the use of `addEventListener`',
+			category: 'Best Practices',
+			recommended: false,
```

Comment:
Do you think this should be shipped as a recommended rule? (It might cause a lot of errors in existing code, though.)
cc: @ota-meshi

Reply:
ultimately i think it should be `recommended`, but im not sure how we can do that without introducing loads of errors like you say 🤔 it'd have to be in the next major i suppose

Reply:
Let’s make it recommended in the next major release.

Reply:
We don't have a mechanism for that yet, but we may be able to include this rule in the recommended set as a warning.

Reply:
Of course we can work on this PR separately.